docs/misc-features.markdown

```diff
@@ -344,3 +344,15 @@ The `hasOneThrough` DSL is built on Joist's `CustomReferences`, so will also wor
 const review = await em.load(BookReview, "1", { author: "publisher" });
 expect(review.author.get.publisher.get.name).toEqual("p1");
 ```
+
+### Cascading Deletions
```

Comment:
Thanks, had forgotten this file even existed

Reply:
Ha, yeah, it is pretty typically for features to show up hear "two or three releases" after they're in.